pl_bolts/models/gans/pix2pix/components.py

```diff
@@ -0,0 +1,167 @@
+import torch
+from torch import nn
+
+
+def center_crop(image, new_shape):
```

Comment:
I think torchvision transforms has `center_crop`, if possible ew can use that perhaps?

Reply:
In my opinion, we already have Bolts heavily depend on torchvision, so we could just use torchvision's one.

@Broda In general, should we have our own implementation in Bolts instead of employing one from major open-source libraries?

Reply:
Thank you for the suggestion! Will make the relevant change.

Reply:
> @Broda In general, should we have our own implementation in Bolts instead of employing one from major open-source libraries?

Sorry @Broda wrong ping 😅

It's for @Borda

Reply:
@Broda I'm very sorry for the wrong ping.

Reply:
Hi @oke-aditya @akihironitta, `PyTorch CenterCrop` requires to pass the desired output size before creating the object. I will have to instantiate the `transforms.CenterCrop` on each Upsample block. Please, let me know what do you think.

Reply:
Ok, I didn't know there is a functional `center_crop` exists in torchvision. It should work.

Reply:
> I think torchvision transforms has `center_crop`, if possible we can use that perhaps?
> Or maybe the idea is to avoid depdence on torchvision ?

@oke-aditya I think we should rely on very major libraries like torchvision instead of having our own implementation because 1) they are (should be) well maintained and tested, and 2) Bolts has already heavy dependency on torchvision anyway. So, in this case, we should use [`torchvision.transforms.functional.center_crop`](https://pytorch.org/vision/stable/transforms.html#torchvision.transforms.functional.center_crop). What's your thought on this?